source/gloperate-glkernel/source/stages/MultiFrameAggregationPipeline.cpp

```diff
@@ -22,7 +23,9 @@ MultiFrameAggregationPipeline::MultiFrameAggregationPipeline(gloperate::Environm
 , renderInterface(this)
 , multiFrameCount("multiFrameCount", this, 64)
 , m_renderFramebufferStage(cppassist::make_unique<gloperate::BasicFramebufferStage>(environment, "BasicFramebufferStage (Renderer)"))
-, m_aggregationFramebufferStage(cppassist::make_unique<gloperate::CustomFramebufferStage>(environment, "CustomFramebufferStage (Accumulation)"))
+, m_aggregationColorTextureStage(cppassist::make_unique<gloperate::TextureStage>(environment, "TextureStage (Color)"))
```

Comment:
In addition, the parentheses ```(``` and ```)``` are problematic, too.
Good names are possible symbols for JavaScript, Python and C++. Starting with a Latin letter followed by alphanumeric characters. Although possible, I would argue to omit underscores in those names as well.